client/resources/SDKTemplate.java

```diff
@@ -0,0 +1,20 @@
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class VersionTemplating {
+
+  public static void main(String[] args) throws IOException {
+    String versionFile = "client/src/main/com/sinch/sdk/SDK.java";
+
+    Stream<String> lines = Files.lines(Paths.get(versionFile));
+    String content = lines.collect(Collectors.joining("\n"));
+    lines.close();
+
+    System.out.println(
+        content.replaceAll("VERSION = \".*\"", String.format("VERSION = \"%s\"", args[0])));
```

Comment:
Would be nice to see `args[0]` extracted into variable with meaningful name, will spare 5 secs of thinking
```suggestion
        content.replaceAll("VERSION = \".*\"", String.format("VERSION = \"%s\"", args[0])));
```

Reply:
Thank you.
This is effectively a better implementation
Done.